tests/pools/test_pool_config.py

```diff
@@ -1,15 +1,16 @@
 # flake8: noqa: E501
 from pathlib import Path
+from tempfile import gettempdir
 
 from blspy import AugSchemeMPL, PrivateKey
 
 from chia.pools.pool_config import PoolWalletConfig
-from chia.util.config import load_config, lock_config, save_config, create_default_chia_config
+from chia.util.config import create_default_chia_config, load_config, lock_config, save_config
 
 
 def test_pool_config():
-    test_root = Path("/tmp")
-    test_path = Path("/tmp/config")
+    test_root = Path(gettempdir())
+    test_path = test_root / "config"
```

Comment:
This is a step forward, but tests should be getting a temporary directory from the `tmp_path` fixture.  This will provide a per-test temporary directory where all of those directories in a given run will be inside the pytest temporary directory for that run which are all inside the overall pytest temporary directory.  We definitely should never have put a hard coded file name directly in `/tmp`, nor even the system temp directory.